qiskit/backends/ibmq/ibmqjob.py

```diff
@@ -564,3 +491,38 @@ def _numpy_type_converter(obj):
     elif isinstance(obj, numpy.ndarray):
         return obj.tolist()
     return obj
+
+
+def _create_job_from_circuit(circuit):
+    """ Helper function that creates a special Job required by the API, from a circuit """
+    job = {}
```

Comment:
This isn't blocking but "Job" in the code tends to indicate something which derives from `BaseJob` which can be confusing. Perhaps we could call api-type job data something else?

Reply:
sure, makes sense.